pkg/mlsstore/store.go

```diff
@@ -149,14 +161,144 @@ func (s *Store) GetIdentityUpdates(ctx context.Context, walletAddresses []string
 func (s *Store) RevokeInstallation(ctx context.Context, installationId []byte) error {
 	_, err := s.db.NewUpdate().
 		Model(&Installation{}).
-		Set("revoked_at = ?", nowNs()).
+		Set("revoked_at = ?", s.nowNs()).
 		Where("id = ?", installationId).
 		Where("revoked_at IS NULL").
 		Exec(ctx)
 
 	return err
 }
 
+func (s *Store) InsertMessage(ctx context.Context, topic string, content []byte) (*messagev1.Envelope, error) {
+	createdAt := s.config.now()
```

Comment:
I'm not sure this ordering is as total as we want. Clock drift between nodes could have newer messages actually end up before older messages.